src/gfxdis/makefile.in

```diff
@@ -15,15 +15,15 @@ MKDIR_P                       = @MKDIR_P@
 .PHONY                        : all install install-strip clean distclean
 
 all                           : $(PROGNAME)
-install                       : | $(bindir)
-	$(INSTALL) -p $(PROGNAME) $(bindir)
+install                       : | $(DESTDIR)/$(bindir)
+	$(INSTALL) -p $(PROGNAME) $(DESTDIR)$(bindir)
 install-strip                 :
-	$(MAKE) INSTALL="$(INSTALL) -s" install
+	$(MAKE) INSTALL="$(INSTALL) -s" DESTDIR=$(DESTDIR) install
 clean                         :
-	rm -f $(PROGNAME)
+	rm -f $(DESTDIR)$(PROGNAME)
```

Comment:
`DESTDIR` should only be used for `install*` and `uninstall*` targets (and `remove-sys` which I should have named `uninstall-sys`). `clean` removes generated files from build directories, not install directories.

Reply:
Done. I have no idea how I let that slip.